src/libraries/System.Threading.Tasks/README.md

```diff
@@ -1,17 +0,0 @@
-# System.Threading.Tasks
```

Comment:
Seems reasonable to remove this, but would like to get ack from @dotnet/area-system-threading-tasks

Reply:
Reached out offline